djconnectwise/migrations/0169_connectwiseboard_time_entry_discussion_flag_and_more.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.0.7 on 2023-03-08 17:09
```

Comment:
I re ran the migrations to tidy things up. I also changed `time_entry_internalanalysis_flag` to `time_entry_internal_analysis_flag`